icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -1691,7 +1710,10 @@ class RemapRule : public SegmentationRule {
             std::terminate();
         }
         remapped = result;
-        U_ASSERT(U_SUCCESS(status));
+        if (!U_SUCCESS(status)) {
```

Comment:
Please change  if (!U_SUCCESS(status))  to if (U_FAILURE(status))  -- at least two of these

Reply:
https://www.youtube.com/watch?v=vC7C4BEvhRs

Done.